examples/Example1.cpp

```diff
@@ -24,11 +24,11 @@
 //
 //
 //  This piece of code was developed as part of ICE-DIP project at CERN.
-//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's 
+//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's
 //  7th Framework programme Marie Curie Actions under grant PITN-GA-2012-316596".
 //
 
-#include "../UMESimd.h"
```

Comment:
I will redo the pull request without adding the extra include. I just think that using `#include <ume/simd>`, `#include <ume/vector>`, etc, would be neat. The main point, however, is that you need to make a clear decision on how you want the installed library files to be organized, preferably following [FHS](http://www.pathname.com/fhs). That's why it's much easier to move headers into `include/`, to make it easier to install them with CMake.

Also, benchmarks, tests, and examples should include the library as user code would, not with relative paths. This is part of why people are confused about how to use your library (see, e.g. issue #47).